From 496b984f889531bf629d77a5fa2211f8cb0a1183 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Wed, 17 Aug 2016 17:18:21 +0200
Subject: [PATCH 4/7] hide error when migrating /var/run to /run

The script tries to copy non-existing files. We simply hide the error

http://bugs.alpinelinux.org/issues/3160
---
 init.d/bootmisc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init.d/bootmisc.in b/init.d/bootmisc.in
index efc1c572..6e473ade 100644
--- a/init.d/bootmisc.in
+++ b/init.d/bootmisc.in
@@ -112,7 +112,7 @@ migrate_to_run()
  		rm $src
  	elif [ ! -L $src -a -d $src ]; then
  		ebegin "Migrating $src to $dst"
- 		cp -a $src/* $dst/
+ 		cp -a $src/* $dst/ 2>/dev/null
  		rm -rf $src
  		eend $?
  	fi
-- 
2.11.0

